tasks/task2.py

```diff
@@ -0,0 +1,22 @@
+# Work with four-digit natural number
+
+input_natural_number = int(input("What is your number?: "))
+# because of int is immutable I need to create list as storage of imputed number
+number = [input_natural_number]
```

Comment:
Is `number` as a list is needed? `number_as_string = str(input_natural_number)` is easier to use